public/js/monitor.js

```diff
@@ -317,6 +317,7 @@ function handleFormSubmits(formEvent) {
     return;
   }
   const thisForm = formEvent.target;
+  thisForm.email.value = thisForm.email.value.trim();
```

Comment:
We should assign this to an `emailValue` variable now that we're using it a few times:

```js
const emailValue = thisForm.email.value.trim();
if (!emailValue || !isValidEmail(emailValue))
```